dashboard/app/app_test.go

```diff
@@ -178,7 +178,7 @@ var testConfig = &GlobalConfig{
 					Config: &EmailConfig{
 						Email:              "bugs@syzkaller.com",
 						DefaultMaintainers: []string{"default@maintainers.com"},
-						SubjectPrefix:      "[syzbot]",
+						SubjectPrefix:      strings.TrimSpace(syzbotSubjectPrefix),
```

Comment:
Did it really make the test file code better?
I think we should disable the linter for this entire file.